onnx_chainer/export_testcase.py

```diff
@@ -0,0 +1,43 @@
+import chainer
+import os
+
+from onnx import numpy_helper
+from onnx_chainer.export import export
+
+
+def export_testcase(model, args, out_dir, graph_name='Graph'):
+    """Export model and I/O tensors of the model in protobuf format.
+
+    Similar to the `export` function, this function first performs a forward
+    computation to a given input for obtaining an output. Then, this function
+    saves the pair of input and output in Protobuf format, which is a
+    defacto-standard format in ONNX.
+
+    This function also saves the model with the name "model.onnx".
+
+    Args:
+        model (~chainer.Chain): The model object.
+        args (list): The arguments which are given to the model
+            directly. Unlike `export` function, only `list` type is accepted.
+        out_dir (str): The directory name used for saving the input and output.
+        graph_name (str): A string to be used for the ``name`` field of the
+            graph in the exported ONNX model.
+    """
+    os.makedirs(out_dir, exist_ok=True)
+    export(model, args,
+           filename='%s/model.onnx' % out_dir,
```

Comment:
please use `os.path.join` function to make file path. L31 is same.

Reply:
Thank you for check. Fixed.